modules/farm_fd2_school/src/entrypoints/FD1/App.vue

```diff
@@ -176,6 +207,18 @@ export default {
       this.unit = null;
       this.comment = '';
     },
+    async fetchCrops() {
+      const fetchedCrops = await fetch(
+        'http://farmos/api/taxonomy_term/plant_type'
+      );
+      const crops = await fetchedCrops.json();
+      // const crop = crops.data[16];
+      // console.log(crop.attributes.name);
+      this.cropList = crops.data;
+    },
+  },
+  async created() {
+    this.fetchCrops();
```

Comment:
The method that you added to fetch the plants is `async`. So it should be called using `await` in your `watch`. 
  - This probably seems to work for now without `await`, but it creates a race condition that will be likely source of bugs in the future.